SignalServiceKit/src/Account/TSAccountManager.m

```diff
@@ -290,11 +304,6 @@ - (void)verifyAccountWithCode:(NSString *)verificationCode
                                                                                forNumber:phoneNumber
                                                                             signalingKey:signalingKey
                                                                                  authKey:authToken];
-    void (^completedRegistrationBlock)() = ^{
-        [self didRegister];
-        [TSSocketManager requestSocketOpen];
```

Comment:
didRegister has been moved until after we're sure the user is able to receive messages.
requestSocketOpen was redundant with the AppDelegate's registrationState observer.